housekeeper/store/api/find.py

```diff
@@ -24,36 +26,49 @@ def bundles(self):
         LOG.info("Fetching all bundles")
         return self.Bundle.query
```

Comment:
We can use the new function here
```suggestion
        return _get_bundle_query()
```